app/models/mdm/web_page.rb

```diff
@@ -1,4 +1,5 @@
 # Web page requested from a {#web_site}.
+require 'webrick'
```

Comment:
You should leave a blank line between requires and the start of the class as requires are a separate code paragraph and a different topic.